src/helper/compare.js

```diff
@@ -29,6 +29,14 @@ const ObjectId = {
 		const idAsString = typeof id === 'string' ? id : String(id);
 		return /^[0-9a-f]{24}$/.test(idAsString);
 	},
+	/**
+	 * Converts a database given id to it's string represenation
+	 * @param {ObjectId|String} id
+	 * @returns {String} string representation of the given object id
+	 */
+	toString(id) {
+		return String(id);
```

Comment:
.toString() is what i propose to use in this situation, because only data types that has this method can convert in a valide way. It make no different for the current situations, but maybe in future.